services/scabbard/cli/man/scabbard-sp-create.1.md

```diff
@@ -0,0 +1,98 @@
+% SCABBARD-SP-CREATE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**scabbard-sp-create** — Creates a Sabre smart permission.
+
+SYNOPSIS
+========
+
+**scabbard sp create** \[**FLAGS**\] \[**OPTIONS**\] ORG_ID NAME
+
+DESCRIPTION
+===========
+This command allows users to create a new Sabre smart permission in state for
+the targeted scabbard service.
```

Comment:
Will this be enough information for someone using this command?  Perhaps add a brief description of what Sabre smart permissions are, or how they're used when scabbard runs a Sabre smart contract.

Reply:
Added a more detailed explanation. It's fairly technical, and we don't have any real uses for smart permissions anywhere yet. It's really here for completeness right now. Also, we are not stabilizing this subcommand until we have a good use for it. We can address this further at that time.

Reply:
Looks great. Thanks!